controllers/subscriptions.go

```diff
@@ -3,6 +3,7 @@ package controllers
 import (
 	"crypto/tls"
 	"encoding/json"
+	"fmt"
```

Comment:
Recommend removing fmt unless you use it for something other than print statements